src/Symfony/Bundle/FrameworkBundle/Tests/Console/Descriptor/TextDescriptorTest.php

```diff
@@ -15,6 +15,11 @@
 
 class TextDescriptorTest extends AbstractDescriptorTest
 {
+    protected function setUp()
+    {
+        putenv('COLUMNS=121');
```

Comment:
destroying it means setting it back to its existing value if any, not removing the env var entirely